lib/parser/onnx/onnx_parser.cc

```diff
@@ -396,7 +396,7 @@ Tensor<T> ONNXParser::ProcessTensor(const onnx::TensorProto& tensor_proto) {
     LOG(ERROR) << "Unsupported external data storage.";
   }
 
-  if (shape.empty() && v.size() > 1) {
+  if (shape.empty() && v.size() >= 1) {
```

Comment:
This will make all scalars to be "[1]" shaped tensor.

Reply:
I'm afraid this will break other cases

Reply:
It is possible to change some input constant parameters from scalar to vector, such as the index of gather.Before scalar processing logic may have to be changed to vector dimension 1